docs/lifecycle.md

```diff
@@ -0,0 +1,17 @@
+## Overview
+
+There are various parameters and functions available to Luigi pertaining the life cycle of listeners, navigation nodes and event data.  
+
+Here is a code sample and a definition of the different parameters:
+
+````
+LuigiClient.addInitListener(initFn)
+LuigiClient.addContextUpdateListener(contextUpdatedFn)
+LuigiClient.getNodeParams()
+LuigiClient.getEventData()
+````
+
+- **addInitListener(initFn)** - Registers a listener that gets called as soon as Luigi is instantiated. Gets called with a context object. Useful to defer your app bootstrap, if you are dependent on authentication data from Luigi.
```

Comment:
1. Turn the bullet points into full sentences.
2. `- **addInitListener(initFn)** registers a listener that is called with a context object as soon as Luigi is instantiated. Defer your application bootstrap if you are dependent on authentication data from Luigi.`